docs/API/components/form-builder.md

```diff
@@ -1,21 +1,80 @@
 # Fliplet Form Builder JS APIs
 
+## Retrieve an instance
+
+### `Fliplet.FormBuilder.get()`
+
+Retries the first or a specific form chartInstances
+
 ```js
 // the "get()" accepts the form name in case you have multiple forms on the page
-Fliplet.FormBuilder.get().then(function (form) {
-  // get the input value
-  form.field('foo').val();
+Fliplet.FormBuilder.get(name)
+  .then(function (form) {
+    // Use form to perform various actions
+  });
+```
+
+## Instance methods
+
+### `.field(name)`
+
+Retrieve a form field.
+
+```js
+Fliplet.FormBuilder.get()
+  .then(function (form) {
```

Comment:
I would personally avoid using the indented `.then` syntax because people might get confused.

```
Fliplet.FormBuilder.get().then(function (form) {

});
``` 
This is simpler to understand